db/migrate/20231110010313_samples_workflow_execution.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Migration to add SamplesWorkflowExecution table
+class SamplesWorkflowExecution < ActiveRecord::Migration[7.1]
+  def change
+    create_table :samples_workflow_executions do |t|
+      t.jsonb :samplesheet_params
```

Comment:
Can you add a default value of an empty hash, see migrations for Attachment or Sample where `metadata` column is defined.

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/308/commits/b93ceec0d2aa3386add4aafbf34c30d56610ee7f